fragments/shared/common/common_no_payload.ks

```diff
@@ -0,0 +1,17 @@
+## common commands without payload configuration ##
```

Comment:
This needs to be kept in sync with smaller subfragments, right?
I guess there is no easy way to do that automatically, should we add comments/notes on it somewhere?
Is the risk of getting out of sync big? Do benefits of having this bigger fragment outweigh it? I guess so.

Reply:
Well, technically it should be perfectly safe™ to run the substitution multiple times once the runtime folder is populated by the fragments and kickstart files.

So you could have a big fragment that refers to some small fragments, which will be resolved on the second pass. On the other hand this adds some complexity and more points where things can fail due to the additional pass/passes.

In any case this is still a huge improvement as at the moment we have basically 150+ copies of the data that we should keep in sync, so reducing that to ~5 places is still good IMHO. :)

So for the time being I guess we can add a comment to the small & big fragments about the need to keep them in sync to not complicate things and do something automatic later on.

Reply:
Agreed